### DISCUSSION TITLE
Enhancement Request: Local Log File

### DISCUSSION THREAD

**Discussion by I-am-PUID-0** [itsToggle/plex_debrid#222] [Ideas]
Hi @itsToggle, 

Would you be willing to add a local log file that echoes the UI output to a file in the directory in your dev roadmap? It may also be useful to make it optional to allow for conserved disk space.

Thx!

**Comment by itsToggle** [comment]
That is a fantastic idea, I should definitly implement that. Thanks!

**Comment by itsToggle** [comment]
Hi I forgot to mention, but this is now implemented :) `/Settings/UI Settings/log to file/`
this will log anything that's printed to your console, as well as any additional debug information into a file called "plex_debrid.log". I'll update the readme with this info aswell

**Reply by I-am-PUID-0** [reply]
Awesome!!! Thanks for being so amenable to requests!!
